logging/logs.rb

```diff
@@ -0,0 +1,19 @@
+#Types of logs:
+#   log.debug "a very nice little debug message"
+#   log.info "things are operating nominally"
+#   log.warn "this is your last warning"
+#   log.error StandardError.new("something went horribly wrong")
+#   log.fatal "I Die!"
+
+require 'logging'
+require 'environment.rb'
+
+# Creates a logfile with the level 'debug'
+#   that appends to log_file.log
+log = Logging.logger['log_file']
```

Comment:
Is there a reason we're putting this in `logging/logs.rb`? I wasn't in the discussion but it feels more intuitive to setup logging in `environment.rb` and write the logs to a file specified in an env var like `LOG_FILE_PATH`.